docs/_articles/en/apex/writing.md

```diff
@@ -3,18 +3,29 @@ title: Write Apex Code
 lang: en
 ---
 
-View outlines of Apex classes and triggers, see code-completion suggestions, and find syntactic errors in your code. The Apex extension is powered by the [Apex Language Server](./en/apex/language-server).
+Use the Apex extension for VS Code to access code-editing features such as code completion, go to definition, view outlines of Apex classes and triggers, refactoring, and find syntactic errors in your code.
```

Comment:
I would recommend keeping this line: The Apex extension is powered by the [Apex Language Server](./en/apex/language-server). This is the only reference to the Apex Language Server in this doc.

Reply:
Deleting the Apex Language Server topic based on Nate and Stephanie's suggestion. So, deleted the reference.